src/components/LegacyEditableHeading/LegacyEditableHeading.module.scss

```diff
@@ -0,0 +1,24 @@
+@import "../../styles/typography";
```

Comment:
All these `LegaclyEditableHeading*` files are the same, the only difference is some imports that were renamed